docs/source/using-executorch-windows.md

```diff
@@ -0,0 +1,149 @@
+```bash
+conda create -yn et python=3.12
+conda activate et
+```
+
+## Check Symlinks
+Set the following environment variable to enable symlinks:
+```bash
+git config --global core.symlinks true
+```
+
+## Set up ExecuTorch
+Clone ExecuTorch from the [official GitHub repository](https://github.com/pytorch/executorch).
+
+```bash
+git clone --recurse -submodules https://github.com/pytorch/executorch.git
+```
+
+## Run the Setup Script
+
+Currently, there are a lot of components that are not buildable on Windows. The below instructions install a very minimal ExecuTorch which can be used as a sanity check.
+
+#### Move into the `executorch` directory
+```bash
+cd executorch
+```
+
+#### (Optional) Run a --clean script prior to running the .bat file.
+```bash
+./install_executorch.bat --clean
+```
+
+#### Run the setup script.
+You could run the .bat file or the python script.
+```bash
+./install_executorch.bat
+# OR
+# python install_executorch.py
+```
+
+## Export MobileNet V2
+
+Create the following script named export_mv2.py
+
+```bash
+from torchvision.models import mobilenet_v2
+from torchvision.models.mobilenetv2 import MobileNet_V2_Weights
+
+mv2 = mobilenet_v2(weights=MobileNet_V2_Weights.DEFAULT) # This is torch.nn.Module
+
+import torch
+from executorch.exir import to_edge
+from executorch.backends.xnnpack.partition.xnnpack_partitioner import XnnpackPartitioner
+
+model = mv2.eval() # turn into evaluation mode
+
+example_inputs = (torch.randn((1, 3, 224, 224)),) # Necessary for exporting the model
+
+exported_graph = torch.export.export(model, example_inputs) # Core Aten graph
+
+edge = to_edge(exported_graph) # Edge Dialect
+
+edge_delegated = edge.to_backend(XnnpackPartitioner()) # Parts of the graph are delegated to XNNPACK
+
+executorch_program = edge_delegated.to_executorch() # ExecuTorch program
+
+pte_path = "mv2_xnnpack.pte"
+
+with open(pte_path, "wb") as file:
+    executorch_program.write_to_file(file) # Serializing into .pte file
+```
+
+### Run the export script to create a `mv2_xnnpack.pte` file.
+
+```bash
+python .\\export_mv2.py
+```
+
+## Build and Install C++ Libraries + Binaries
+```bash
+del -Recurse -Force cmake-out; `
+cmake . `
+  -DCMAKE_INSTALL_PREFIX=cmake-out `
+  -DPYTHON_EXECUTABLE=C:\Users\nikhi\miniconda3\envs\et\python.exe `
+  -DCMAKE_PREFIX_PATH=C:\Users\nikhi\miniconda3\envs\et\Lib\site-packages `
+  -DCMAKE_BUILD_TYPE=Release `
+  -DEXECUTORCH_BUILD_EXTENSION_TENSOR=ON `
+  -DEXECUTORCH_BUILD_FLATC=ON `
+  -DEXECUTORCH_BUILD_PYBIND=OFF `
+  -DEXECUTORCH_BUILD_XNNPACK=ON `
+  -DEXECUTORCH_BUILD_KERNELS_CUSTOM=ON `
+  -DEXECUTORCH_BUILD_KERNELS_OPTIMIZED=ON `
+  -DEXECUTORCH_BUILD_KERNELS_QUANTIZED=ON `
+  -DEXECUTORCH_ENABLE_LOGGING=ON `
+  -T ClangCL `
+  -Bcmake-out; `
+cmake --build cmake-out -j64 --target install --config Release
+```
+
+## Run Mobilenet V2 model with XNNPACK delegation
+
+```bash
+.\cmake-out\backends\xnnpack\Release\xnn_executor_runner.exe --model_path=.\mv2_xnnpack.pte
```

Comment:
Can you escape the backslashes (replace \ with \\, I believe) here? It renders weirdly in the docs due to interpreting as a escape char - see https://docs-preview.pytorch.org/pytorch/executorch/11273/using-executorch-windows.html.